Core/CoreTests/Planner/Model/UseCase/GetPlannablesTests.swift

```diff
@@ -217,7 +217,7 @@ class GetPlannablesTests: CoreTestCase {
     }
 
     func testWriteCalendarEvents() {
-        let response = GetPlannables.Response.make(calendarEvents: [
+        let response = GetPlannables.Response.init(calendarEvents: [
```

Comment:
```suggestion
        let response = GetPlannables.Response(calendarEvents: [
```

Same for the other one below